apps/meteor/app/lib/server/methods/sendMessage.ts

```diff
@@ -16,7 +18,12 @@ import { MessageTypes } from '../../../ui-utils/server';
 import { sendMessage } from '../functions/sendMessage';
 import { RateLimiter } from '../lib';
 
-export async function executeSendMessage(uid: IUser['_id'], message: AtLeast<IMessage, 'rid'>, previewUrls?: string[]) {
+export async function executeSendMessage(
+	uid: IUser['_id'],
+	message: AtLeast<IMessage, 'rid'>,
+	previewUrls?: string[],
+	filesArray?: Partial<IUpload>[],
```

Comment:
Try to not use `Array` on field names, it's better to use just `files`. In this case you should receive only the ids of the files to confirm, they were already uploaded, so a better name would be `uploadIdsToConfirm`

Reply:
done changed the name to `uploadIdsToConfirm`